### DISCUSSION TITLE
How to start Interstellar on cyclic

### DISCUSSION THREAD

**Discussion by Globalfun32y439y54** [UseInterstellar/Interstellar#708] [Q&A]
I can't get to Interstellar on cyclic. How do I port forward?

**Comment by nvd123YT** [comment]
cyclic shut down

**Reply by Globalfun32y439y54** [reply]
What do you mean?

**Reply by nvd123YT** [reply]
It isn't accepting new signups at least. Maybe it works for those that already have an account

**Reply by nvd123YT** [reply]
I haven't checked in a bit tho so I don't remember

**Comment by nvd123YT** [comment]
Yes it did shut down I checked

Use Vercel, Koyeb, Render, or Glitch
